### DISCUSSION TITLE
Update email in public user whenever email update in auth.user?

### DISCUSSION THREAD

**Discussion by mohitxskull** [supabase/supabase#6623] [Questions]
i wan't to know these and some resources to learn plsql, so i get to know how to delete, insert and update

**Comment by GaryAustin1** [comment]
So I assume you have done something like this:
https://supabase.com/docs/guides/auth/managing-user-data#advanced-techniques

If so, then you would create a similar function doing an update to your public user table and call it with an after update trigger.

You can also have a trigger on both insert and update and use if statements to decide what to do: https://www.postgresql.org/docs/current/plpgsql-trigger.html
That same document also gives examples of doing inserts/updates/deletes in a function.
